app/src/main/java/com/codesoom/HttpEnum/HttpStatusCode.java

```diff
@@ -0,0 +1,26 @@
+package com.codesoom.HttpEnum;
+
+/**
+ * HTTP 상태코드를 정의합니다.
+ * @link https://datatracker.ietf.org/doc/html/rfc7231#section-6
+ */
+public enum HttpStatusCode {
+
+    OK(200),
+    NOTFOUND(404),
+    CREATED(201),
+    NOCONTENT(204),
+    BADREQUEST(400);
```

Comment:
이런건 읽기 좋게 이렇게 `_`를 넣어줍시다.
```suggestion
    NOT_FOUND(404),
    CREATED(201),
    NO_CONTENT(204),
    BAD_REQUEST(400);
```

Reply:
보기 쉽도록 _를 넣었습니다!

Reply:
잘 하셨어요!